### ISSUE TITLE
Support projects without requiring write-access

### ISSUE THREAD

**Issue by sjoerdtalsma** [dependabot/feedback#22]
Since _dependabot_  already creates pull requests, it could do so in its own `dependabot` space.
I tried the bot just now on my Java project and absolutely love it, but will remove it again because it requires **write** access to the repository.

Instead I think the :robot: should behave like any decent open source user and create pull-requests from its own repository. Giving out write access should not be necessary in my opinion.

Thanks for creating this useful tool though! :+1:

**Comment by greysteil** [comment]
Thanks for the feedback!

Unfortunately Dependabot is between a rock and a hard place on this one - for private repos we really don’t want to clone your repo and create PRs from a fork (there are security concerns, and it’s not the expected flow). GitHub’s app permissions are one-size-fits-all, so while it would be ideal for us to treat public and private projects differently there’s no straightforward way for us to ask for different permissions for them :-(

I’ve been in touch with the good folks at GitHub about the vibe asking them for a fix, but in the meantime the current setup is the best we can do 🙁

**Comment by sjoerdtalsma** [comment]
>GitHub’s app permissions are one-size-fits-all, so while it would be ideal for us to treat public and private projects differently there’s no straightforward way for us to ask for different permissions for them :-(

I see. Maybe create a `dependabot-os` variant that clones repo's and keep the current bot as it is?
I agree that is an unwanted situation though that _really_ should be handled by GitHub. Is there any ticket I can upvote? :wink:

**Comment by ottlinger** [comment]
Unfortunately requiring write-access prevents from using this great service with projects from the Apache Software Foundation (ASF) ..... is there a change to reconsider? Or being able to configure using dependabot without auto-merge instead?

https://issues.apache.org/jira/browse/INFRA-18306

**Comment by greysteil** [comment]
@ottlinger still in talks with GitHub about this - we should have something for you by the end of the month. Would love to have the ASF using Dependabot!

**Comment by sjoerdtalsma** [comment]
@greysteil 
>we should have something for you by the end of the month

Please keep us posted !

**Comment by ottlinger** [comment]
@greysteil I just read that Github announced you were bought by them .... is that the surprise that might allow a readonly integration for ASF :-) ?

German only:
https://www.heise.de/newsticker/meldung/GitHub-stellt-Funktionen-fuer-Sicherheit-und-Open-Source-vor-4430470.html

**Comment by greysteil** [comment]
Yes! I was hoping that now Dependabot is owned by GitHub you'd be happier to use it with write access. What do you reckon? The medium term trajectory for Dependabot is now for it to be fully integrated with GitHub, and configured from within GitHub.
